<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TileCreatorClass</class>
 <widget class="QMainWindow" name="TileCreatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TileCreator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QComboBox" name="cmbTileCreator">
        <item>
         <property name="text">
          <string>影像切片</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>高程切片</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>矢量切片</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbTileflag">
        <item>
         <property name="text">
          <string>单个文件</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>多个文件</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="ptnres">
        <property name="text">
         <string>原始数据路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ledresfile"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>瓦片最小级别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="text">
         <string>0</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>瓦片最大级别：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ledMaxlev"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QPushButton" name="ptnobj_2">
        <property name="text">
         <string>瓦片扩展名：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmbext">
        <item>
         <property name="text">
          <string>png</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>tif</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>json</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="ptnobj">
        <property name="text">
         <string>瓦片保存路径：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ledobjfile"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="ptnbegin">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>168</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="tilecreator.qrc"/>
 </resources>
 <connections/>
</ui>
